### ISSUE TITLE
[15] Text alternative contains unnecessary information

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#288]
Applies: Images for Leadership on the Homepage
Type: WCAG best practice – related to 1.1.1 Non-Text Content
Platform: Desktop and mobile 
Severity: Low

The text alternative for the images with the Leadership section includes extraneous information that is not necessary. The alt text for the image of Hon Angus Taylor MP is ‘Image of Hon Angus Taylor MP’.
Screen readers inform the user that the object is an image, so alternative text of "image of..." (and similar) should be avoided as it is duplicated information.
For the example below, JAWS says: “Graphic. Image of the Hon Angus Taylor MP”.

**Recommendation**
Do not include words such as “image of” or “graphic of” with the alt text.

Removing “image of” would result in JAWS reading: “Graphic. The Hon Angus Taylor MP”.

We’d also suggest using the ‘Honourable’ instead of ‘Hon’ as it is better pronounced by screen readers.

**Comment by bjgordon** [comment]
It would be good if guides included some guidance on alt text. cc @joolswood

**Comment by joolswood** [comment]
This is the guidance I was mentioning @bjgordon http://content-guide.apps.staging.digital.gov.au/accessibility-inclusivity/#images-alt-text We're likely to develop this a little bit, and also will improve the guidance in the Design Guide.

BTW I was going to discuss this with you @AndrewArch - @klepas commented that we should recommend CSS image over a null alt text for decoration image. I was thinking of including this, but noting users of the Content Guide are less likely to have access to CSS than users of the Design Guide.

**Comment by joolswood** [comment]
Related to https://github.com/govau/content-guide/issues/20

**Comment by bjgordon** [comment]
Thanks @joolswood I had missed that, looks good.

**Comment by joolswood** [comment]
Here we go https://github.com/govau/content-guide/commit/d75188defdc32d878bdc7e3d73177146cebeec4b#diff-94c8e3b8ca80a0a57dddc677c021b6daR54
